### DISCUSSION TITLE
An Upcoming Clean Progress Bar in knitr | /en/2023/01/knitr-progress-bar/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1463] [Comments]
Do you love the multi-line progress bar when knitting a document? You probably don&rsquo;t. Neither do I. It&rsquo;s too verbose and clutters the console with too much information that often has too &hellip;

https://yihui.org/en/2023/01/knitr-progress-bar/

**Comment by vitu1** [comment]
Hi Yihui,

This update is great and cleans up the word soup nicely during knitting. One issue that I've encountered is that I no longer see my messages being sent to stderr during knitting, only the progress bar. Is there an option to reenable message outputs during knitting with this single-line progress bar?

Thank you,

**Reply by yihui** [reply]
Yes, please use `message = NA`: https://github.com/yihui/yihui.org/discussions/1458#discussioncomment-4463398

**Reply by vitu1** [reply]
Thank you! This fixed the issue
